azure-pipelines-richnav.yml

```diff
@@ -0,0 +1,27 @@
+# Run on master CI builds
+trigger:
+- master
+
+pr:
+- master # for testing, will remove before merging
+
+jobs:
+- job: RichCodeNav_Indexing
+  pool:
+    name: NetCorePublic-Pool
+    queue: BuildPool.Windows.10.Amd64.VS2017.Open
```

Comment:
Any reason this needs the VS2017 machine image versus just BuildPool.Windows.10.Amd64.Open?

Reply:
I think the indexing will just need msbuild tools so will remove that!